Dockerfile

```
@@ -11,7 +11,7 @@ FROM golatest AS go-darwin
 FROM golatest AS go-windows-amd64
 FROM golatest AS go-windows-386
 FROM golatest AS go-windows-arm
-FROM --platform=$BUILDPLATFORM tonistiigi/golang:497feff1-${BASE_VARIANT} AS go-windows-arm64
+FROM --platform=$BUILDPLATFORM golang:1.17beta1-${BASE_VARIANT} AS go-windows-arm64
```

I see https://github.com/golang/go/issues/46815 was closed; can you try updating to rc1? 

```suggestion
FROM --platform=$BUILDPLATFORM golang:1.17rc1-${BASE_VARIANT} AS go-windows-arm64
```